tests/models/covid19/BaseSpendingByCfdaRow-test.js

```diff
@@ -10,53 +10,60 @@ const row = Object.create(BaseSpendingByCfdaRow);
 row.populate(mockCfdaData);
 
 describe('COVID-19 spending by CFDA row', () => {
-    describe('name column properties', () => {
-        it('should store the id', () => {
-            expect(row._id).toEqual('43');
+    describe('CoreSpendingTableRow properties', () => {
+        describe('name column properties', () => {
+            it('should store the id', () => {
+                expect(row._id).toEqual('43');
+            });
+            it('should store the code', () => {
+                expect(row._code).toEqual('090');
+            });
+            it('should store the description', () => {
+                expect(row.description).toEqual('Description text');
+            });
         });
-        it('should store the code', () => {
-            expect(row._code).toEqual('090');
+        describe('obligation properties', () => {
+            it('should store the raw obligated amount', () => {
+                expect(row._obligation).toEqual(89000000.01);
+            });
+            it('should store the formatted obligated amount', () => {
+                expect(row.obligation).toEqual('$89,000,000');
+            });
         });
-        it('should store the description', () => {
-            expect(row._description).toEqual('Description text');
+        describe('outlay properties', () => {
+            it('should store the raw outlayed amount', () => {
+                expect(row._outlay).toEqual(70000000.98);
+            });
+            it('should store the formatted outlayed amount', () => {
+                expect(row.outlay).toEqual('$70,000,001');
```

Comment:
I like this pattern a lot more than what we're doing w/ award amounts, which is much more verbose.